.github/workflows/release-helm-charts.yaml

```diff
@@ -0,0 +1,33 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Fetch history
+        run: git fetch --prune --unshallow
```

Comment:
Not sure if we really need this but 🤷

Reply:
@regadas read it https://github.com/actions/checkout

> Only a single commit is fetched by default, for the ref/SHA that triggered the workflow. Set fetch-depth: 0 to fetch all history for all branches and tags. Refer [here](https://help.github.com/en/articles/events-that-trigger-workflows) to learn which commit $GITHUB_SHA points to for different events.

Reply:
Right, I knew about that 😄 I wasn't sure why the helm action would require it; but they do look at the tags.

Reply:
Indeed. That is needed so that we can fetch all tags/branches. I just also saw that checkout-action has a more claver way to do it: https://github.com/actions/checkout#fetch-all-history-for-all-tags-and-branches -- so feel free to opt-in for: 
```yaml
- uses: actions/checkout@v3
  with:
    fetch-depth: 0
```